### ISSUE TITLE
Need the PlatformIO device examples ported to the new doc set in some fashion

### ISSUE THREAD

**Issue by leroyle** [helium/docs#139]
It would be good to have the PlatformIO examples incorporated into the new doc set.
https://github.com/helium/longfi-platformio

**Comment by kent-williams** [comment]
Hi @leroyle,

I agree! It would be great to see guides for each example. If you'd like to mimic the style of the Arduino guides, any contributions will be welcome!

**Comment by leroyle** [comment]
@kent-williams 
I can shake some time loose I think,
I did some work in the longfi-platformio repo, where I added a couple more devices. There I took the tac of a single top level PlatformIO install/setup/usage guide, expanding on what was already there.  In that single guide I had pointed out potential device differences, with said differences being documented under the specific device in it's README. The thought there was if something within the PIO install/setup changed, external links for example, we would only need to change it in one place.

For the Arduino docs, they are each complete, self contained. Each device has the full install/setup directions for getting the IDE up and running and app code uploaded to the device. 

So, which way are you thinking for the PlatformIO examples, and where should they hook into the existing (new) docs?
